doc/conf.py

```diff
@@ -1,4 +1,5 @@
 from sphinx_mdolab_theme.config import *
+import urllib.request
```

Comment:
What is using this package?

Reply:
Whoops I had a previous version that was using this to fetch `mdolab.bib` but eventually decided to maintain a separate file, since many references don't exist in mdolab.bib. Will remove.